CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+
+## [1.30.1](https://github.com/brave/brave-ios/releases/tag/v1.30.1)
+
+- Reordered "Pull-to-Refresh" setting position to be above "Set default browser" option for iOS 14 devices. ([#4045](https://github.com/brave/brave-ios/issues/4045))
+- Fixed tab webView(page snapshot) from showing up when opening or closing a tab via tab tray. ([#4019](https://github.com/brave/brave-ios/issues/4019))
```

Comment:
quick nit: there's a space missing between `webView(page snapshot)`. But I think we can do something like this where it actually mentions previously viewed websites being displayed rather than `snapshot`. I don't think some users will understand what we mean by a `snapshot`. Thoughts?

```
- Fixed previously viewed websites from being displayed in the tab webView when opening or closing a tab via the tab tray.
```